src/sdk/sdk.cpp

```diff
@@ -36,8 +55,67 @@ namespace {
 
 namespace sdk {
     namespace {
-        __forceinline void PrintClassInfo(codegen::generator_t::self_ref builder, std::int16_t alignment, std::int16_t size) {
-            builder.comment(std::format("Alignment: {}", alignment)).comment(std::format("Size: {:#x}", size));
+        __forceinline void PrintClassInfo(codegen::generator_t::self_ref builder, SchemaClassInfoData_t* class_info) {
+            builder
+                .comment(std::format("Registered binary: {} (project '{}')", sdk::g_schema->GetClassInfoBinaryName(class_info),
+                                     sdk::g_schema->GetClassProjectName(class_info)))
+                .comment(std::format("Alignment: {}", class_info->m_align_of))
+                .comment(std::format("Size: {:#x}", class_info->m_size));
+            
+            if ((class_info->m_class_flags & SCHEMA_CF1_HAS_VIRTUAL_MEMBERS) != 0) // @note: @og: its means that class probably does have vtable
+                builder.comment("Is Polymorphic");
```

Comment:
Instead of writing some abstract info that nobody would understand, just write exactly what it means
```suggestion
                builder.comment("Has vtable");
```

Reply:
+